action.yml

```diff
@@ -30,9 +30,19 @@ inputs:
     description: 'remote target username'
     required: true
     default: 'root'
-# outputs:
-#   output: # id of output
-#     description: 'the output of the command'
+  PORT:  # id of input
+    description: 'ssh port'
+    required: false
```

Comment:
All of these are not required, and so won't interfere with existing usage of the action